src/providers/postgres/qgspostgresconn.cpp

```diff
@@ -1048,10 +1049,17 @@ QString QgsPostgresConn::postgisVersion() const
   // apparently PostGIS 1.5.2 doesn't report capabilities in postgis_version() anymore
   if ( mPostgisVersionMajor > 1 || ( mPostgisVersionMajor == 1 && mPostgisVersionMinor >= 5 ) )
   {
-    result = PQexec( QStringLiteral( "SELECT postgis_geos_version()" ) );
+    result = PQexec( QStringLiteral( "SELECT postgis_geos_version(), postgis_proj_version()" ) );
     mGeosAvailable = result.PQntuples() == 1 && !result.PQgetisnull( 0, 0 );
+<<<<<<< HEAD
     QgsDebugMsg( QStringLiteral( "geos:%1 proj:%2" )
                  .arg( mGeosAvailable ? result.PQgetvalue( 0, 0 ) : "none" ) );
+=======
```

Comment:
```suggestion
```

Reply:
@m-kuhn , can you help me with this? I don't have commit rights to this repo. Above are the changes needed.